### PR TITLE
add counts to address tabs

### PR DESCRIPTION
**Pull request by isstuev**
resolve https://github.com/blockscout/frontend/issues/1055

### CODE REVIEW

File: types/api/address.ts [hunk 1]
```diff
@@ -12,6 +12,7 @@ export interface Address extends UserTags {
   creator_address_hash: string | null;
   creation_tx_hash: string | null;
   exchange_rate: string | null;
+  // TODO: if we are happy with tabs-counters method, should we delete has_something fields?
```
Comment (tom2drum) [inline types/api/address.ts:15]: let's do it

File: ui/shared/Tabs/TabCounter.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  return (
+    <Text
+      color={ count > 0 ? 'text_secondary' : zeroCountColor }
+      ml={ 1 }
+      sx={ sx }
+      transitionProperty="color"
+      transitionDuration="normal"
+      transitionTimingFunction="ease"
+    >
+      { count > 50 ? '50+' : count }
```
Comment (tom2drum) [inline ui/shared/Tabs/TabCounter.tsx:32]: maybe, I would leave a comment about "magic 51 number" 😄

File: ui/shared/Tabs/TabCounter.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  const zeroCountColor = useColorModeValue('blackAlpha.400', 'whiteAlpha.400');
+
+  if (count === undefined) {
+    return null;
+  }
+
+  const sx: SystemStyleObject = {};
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore:
+  sx[`.${ parentClassName }:hover &`] = { color: 'inherit' };
```
Comment (tom2drum) [inline ui/shared/Tabs/TabCounter.tsx:21]: 🙈

**Review by tom2drum — APPROVED** [review]

File: types/api/address.ts (hunk 1 above)
Comment (isstuev) [reply types/api/address.ts:15]: later after tests

File: ui/shared/Tabs/TabCounter.tsx (hunk 3 above)
Comment (isstuev) [reply ui/shared/Tabs/TabCounter.tsx:21]: idk how I can make ts happy here

Comment (tom2drum) [reply ui/shared/Tabs/TabCounter.tsx:21]:
```
const sx: SystemStyleObject = {
    [`.${ parentClassName }:hover &`]: { color: 'inherit' },
};
```

have you tried this? works for me

**Pull request merged**